java/arcs/sdk/Utils.kt

```diff
@@ -12,11 +12,16 @@
 package arcs.sdk
 
 /**
- * Utilities for ease-of-use
- *
- * Sugar to delegate function calls to methods on the current [RuntimeClient].
+ * Utilities for ease-of-use. Implemented for each platform.
  */
+expect object Utils {
+    fun log(msg: String)
+
+    fun abort()
+
+    fun assert(message: String, cond: Boolean)
```

Comment:
This could be slightly more flexible inverting the order of the parameters, as an additional function could be created with the signature:

```kotlin
inline fun assert(condition: Boolean, crossinline messageBuilder: () -> String)
```

This variant would allow the message to perform string interpolation when building the message _only_ when the condition fails, rather than always having to do it regardless of the success/failure.